domains/sa.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "shz-x",
+    	"email": "shz5393@gmail.com"
+
+    },
+    "records": {
+        "A": "129.213.100.223"
```

A records is an instance of Array
```suggestion
        "A": ["129.213.100.223"]
```